### PR TITLE
Carrington makefile

### PR DESCRIPTION
**Pull request by ursg**
Binaries built with this makefile compile and run on the new carrington cluster.
This might still benefit from some flag tweaking for performance, but works.

Note the use of the ```-flto``` flag for link time optimization, so the linking step takes quite a while.

### CODE REVIEW

File: MAKE/Makefile.carrington_gcc_openmpi [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+MPT_BRAND = OpenMPI
+MPT_VERSION = 4.0.5
+
+JEMALLOC_VERSION = 5.2.1
+LIBRARY_PREFIX_B = /proj/ykempf/libraries
+LIBRARY_PREFIX = /proj/group/spacephysics/libraries
+
+#compiled libraries mostly in modules
+LIB_PROFILE = -L$(LIBRARY_PREFIX_B)/$(CC_BRAND)/$(CC_BRAND_VERSION)/$(MPT_BRAND)/$(MPT_VERSION)/phiprof/lib -lphiprof -Wl,-rpath=$(LIBRARY_PREFIX_B)/$(CC_BRAND)/$(CC_BRAND_VERSION)/$(MPT_BRAND)/$(MPT_VERSION)/phiprof/lib
```
Comment (ykempf) [inline MAKE/Makefile.carrington_gcc_openmpi:77]: Can that be written in two lines to avoid the long duplicate stuff?

**Review by ykempf — COMMENTED** [review]
Here are my comments. :)

**Comment by markusbattarbee** [comment]
So are you running and compiling with or without the UCX/1.9.0-GCCcore-10.2.0  module?

**Comment by ykempf** [comment]
```
Currently Loaded Modules:
  1) GCCcore/10.2.0               3) binutils/2.35-GCCcore-10.2.0   5) numactl/2.0.13-GCCcore-10.2.0   7) libxml2/2.9.10-GCCcore-10.2.0      9) hwloc/2.2.0-GCCcore-10.2.0      11) UCX/1.9.0-GCCcore-10.2.0         13) PMIx/3.1.5-GCCcore-10.2.0
  2) zlib/1.2.11-GCCcore-10.2.0   4) GCC/10.2.0                     6) XZ/5.2.5-GCCcore-10.2.0         8) libpciaccess/0.16-GCCcore-10.2.0  10) libevent/2.1.12-GCCcore-10.2.0  12) libfabric/1.11.0-GCCcore-10.2.0  14) OpenMPI/4.0.5-GCC-10.2.0
```
it comes out of
```
module load GCC/10.2.0
module load OpenMPI/4.0.5-GCC-10.2.0
```

**Comment by ursg** [comment]
I have now been succesfully running ionosphere tests on carrington with exactly this setup (and the modules as listed on the top). I would suggest we'd just merge this then.

Any improvements (like a working PAPI) can always be merged on top.

**Pull request merged**